nixnet/_session/frames.py

```diff
@@ -22,6 +22,25 @@ def __repr__(self):
     def _create_item(self, handle, index, name):
         return Frame(handle, index, name)
 
+    @property
+    def payld_len_max(self):
```

Comment:
How did this move not break our tests?

https://github.com/ni/nixnet-python/blob/master/tests/test_frames.py#L216